src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -132,6 +132,7 @@ export default class GroupComparisonMutationsTab extends React.Component<
                     </div>
                     <GroupComparisonMutationsTabPlot
                         store={this.props.store}
+                        urlWrapper={this.props.urlWrapper}
```

Comment:
it would be better to give this component a property onScaleToggle and then update the urlWrapper at this level.